### PR TITLE
`lib/downloader`: Add an option for disabling progress bar

### PR DESCRIPTION
**Pull request by supechicken**
This allows the user to hide the download progress bar by setting `CREW_DOWNLOADER_SHOW_PROGBAR` to `0`, which is useful when redirecting output to log files (when outputting to log files, the `\r` control char will be ignored by text editors and show many lines of progress bar)

Tested on `x86_64`

### CODE REVIEW

File: lib/downloader.rb [hunk 1]
```diff
@@ -133,15 +133,17 @@ def http_downloader (url, filename = File.basename(url), verbose = false)
...
-          downloaded_size += chunk.size # record downloaded size, used for showing progress bar
-          if file_size.positive?
-            # calculate downloading progress percentage with the given file size
-            percentage = (downloaded_size / file_size) * 100
-            # show progress bar, file size and progress percentage
-            printf "\r""[%-#{@progBarW}.#{@progBarW}s] %9.9s %3d%%",
-                   '#' * ( @progBarW * (percentage / 100) ),
-                   human_size(file_size),
-                   percentage
+          if CREW_DOWNLOADER_SHOW_PROGBAR
```
Comment (uberhacker) [inline lib/downloader.rb:136]: `unless CREW_HIDE_PROGBAR`?

File: lib/const.rb [hunk 2]
```diff
@@ -89,6 +89,8 @@
 
 # Downloader maximum retry count
 CREW_DOWNLOADER_RETRY = ( ENV['CREW_DOWNLOADER_RETRY'].to_s.empty? ) ? 3 : ENV['CREW_DOWNLOADER_RETRY'].to_i
+# Downloader show progress bar or not
+CREW_DOWNLOADER_SHOW_PROGBAR = ENV.fetch('CREW_DOWNLOADER_SHOW_PROGBAR', '1').eql?('1')
```
Comment (uberhacker) [inline lib/const.rb:93]: Shouldn't this default to 1?  The initial value will be nil (ie. same as zero or false) if the environment variable is not set.  Maybe you can change to `CREW_HIDE_PROGBAR` instead?

File: lib/downloader.rb (hunk 1 above)
Comment (supechicken) [reply lib/downloader.rb]: Done

**Comment by uberhacker** [comment]
Bump crew version in `lib/const.rb`?

**Comment by supechicken** [comment]
Done

**Pull request merged**